core/steps/microvm/start.go

```diff
@@ -13,16 +14,22 @@ import (
 	"github.com/weaveworks/flintlock/pkg/planner"
 )
 
-func NewStartStep(vm *models.MicroVM, vmSvc ports.MicroVMService) planner.Procedure {
+func NewStartStep(
+	vm *models.MicroVM,
+	vmSvc ports.MicroVMService,
+	bootTime int,
```

Comment:
nit: maybe `bootWaitTimeSeconds` or something more descriptive.....just idea and feel free to ignore.